### ISSUE TITLE
`experimental_use_coverage_metadata_files`

### ISSUE THREAD

**Issue by UebelAndre** [bazelbuild/rules_rust#2082]
`experimental_use_coverage_metadata_files` is a flag to have coverage tooling added as [coverage_common.instrumented_files_info.metadata_files](https://bazel.build/versions/6.3.0/rules/lib/coverage_common?hl=en#instrumented_files_info) (taking advantage of https://github.com/bazelbuild/bazel/issues/18697) instead of reporting tools like `llvm-cov` and `llvm-profdata` as runfiles to each test. To enable this flag pass the following:
```
--@rules_rust//rust/settings:experimental_use_coverage_metadata_files
```
